spinnaker/spinnaker_testing/gcp_quota_check.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""GCP utility to check quotas for a given project.
+
+Given a GCP project and a quota threshold percentage, checks that
+all the quotas are under that threshold percentage, else fails.
+
+This script is a temporary workaround until we get more solid
+quota notification support.
+"""
+
+import argparse
+import logging
+import sys
+
+import citest.gcp_testing as gcp
+
+PLATFORM_FULL_SCOPE = 'https://www.googleapis.com/auth/cloud-platform'
+
+def parse_args():
+  parser = argparse.ArgumentParser(description='Check GCP quotas.')
+  parser.add_argument('--gcp_project', help='Your GCP project.')
+  parser.add_argument('--quota_threshold',
+                      type=float,
+                      help='Percentage of use your quota should be under.')
+  parser.add_argument('--credentials_path',
+                      help='Path to your GCP service account key.')
+  args = parser.parse_args()
+  if (not args.quota_threshold
+      or not args.gcp_project
+      or not args.credentials_path):
+    logging.error('Invalid values for input args: must not be None.')
+    raise ValueError('Invalid values for input args: must not be None.')
```

Comment:
Given that this is in citest, I recommend hooking into Eric's existing logging/verification tools